plutus-tx-plugin/test/size/Main.hs

```diff
@@ -4,167 +4,245 @@
 
 module Main (main) where
 
-import PlutusTx.Code (CompiledCode)
+import Data.Kind (Type)
+import Data.Tagged (Tagged (Tagged))
+import Data.Typeable (Typeable)
+import PlutusTx.Code (CompiledCode, sizePlc)
 import PlutusTx.IsData.Class (fromBuiltinData, toBuiltinData, unsafeFromBuiltinData)
 import PlutusTx.Prelude qualified as Plutus
 import PlutusTx.Ratio qualified as PlutusRatio
 import PlutusTx.Test
 import PlutusTx.TH (compile)
 import Prelude
-import Test.Tasty (defaultMain, testGroup)
+import Test.Tasty (TestName, TestTree, defaultMain, testGroup)
 import Test.Tasty.Extras (runTestNested, testNested)
+import Test.Tasty.Providers (IsTest (run, testOptions), singleTest, testFailed, testPassed)
 
 main :: IO ()
-main = defaultMain $ testGroup "Size regression tests"
-  [ runTestNested ["test", "size", "Golden"]
-      [ testNested "Rational"
-          [ testNested "Eq"
-              [ goldenSize "equal" ratEq
-              , goldenSize "not-equal" ratNeq
-              ]
-          , testNested "Ord"
-              [ goldenSize "compare" ratCompare
-              , goldenSize "less-than-equal" ratLe
-              , goldenSize "greater-than-equal" ratGe
-              , goldenSize "less-than" ratLt
-              , goldenSize "greater-than" ratGt
-              , goldenSize "max" ratMax
-              , goldenSize "min" ratMin
-              ]
-          , testNested "Additive"
-              [ goldenSize "plus" ratPlus
-              , goldenSize "zero" ratZero
-              , goldenSize "minus" ratMinus
-              , goldenSize "negate-specialized" ratNegate
-              ]
-          , testNested "Multiplicative"
-              [ goldenSize "times" ratTimes
-              , goldenSize "one" ratOne
-              , goldenSize "scale" ratScale
-              ]
-          , testNested "Serialization"
-              [ goldenSize "toBuiltinData" ratToBuiltin
-              , goldenSize "fromBuiltinData" ratFromBuiltin
-              , goldenSize "unsafeFromBuiltinData" ratUnsafeFromBuiltin
-              ]
-          , testNested "Construction"
-              [ goldenSize "unsafeRatio" ratMkUnsafe
-              , goldenSize "ratio" ratMkSafe
-              , goldenSize "fromInteger" ratFromInteger
-              ]
-          , testNested "Other"
-              [ goldenSize "numerator" ratNumerator
-              , goldenSize "denominator" ratDenominator
-              , goldenSize "round" ratRound
-              , goldenSize "truncate" ratTruncate
-              , goldenSize "properFraction" ratProperFraction
-              , goldenSize "recip" ratRecip
-              , goldenSize "abs-specialized" ratAbs
+main =
```

Comment:
I went through this whole function trying to figure out what changed, only to realize it is just formatting. Please avoid mixing formatting with other changes.